### PR TITLE
Support for Activity Stream added

### PR DESCRIPTION
**Pull request by ngoyal16**
## what
* Allow to create Activity Stream for RDS Cluster

Added by @nuru:
* Move PostgreSQL testing to proper test region (us-east-2)
* Fix expected subnet CIDR due to fewer availability zones in us-east-2 than us-east-1
* Increase test parallelism so that tests complete in under 60 minutes

## why
* Can be used to enable and activity stream from the cluster module itself

### CODE REVIEW

**Comment by goruha** [comment]
@ngoyal16 Thanks for the PR.
Can you also add outputs for stream
`kinesis_stream_name` and `id`

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster_activity_stream#attributes-reference

**Review by goruha — CHANGES_REQUESTED** [review]
Needs activity stream outputs

**Comment by goruha** [comment]
@ngoyal16 Tests failed 

```
╷
│ Error: Missing resource instance key
│ 
│   on main.tf line 401, in resource "aws_rds_cluster_activity_stream" "primary":
│  401:   resource_arn = aws_rds_cluster.primary.arn
│ 
│ Because aws_rds_cluster.primary has "count" set, its attributes must be
│ accessed on specific instances.
│ 
│ For example, to correlate with indices of a referring resource, use:
│     aws_rds_cluster.primary[count.index]
```

**Comment by ngoyal16** [comment]
@goruha i have added the outputs. also fix the issue related to cluster arn

**Comment by ngoyal16** [comment]
@jamengual or @goruha can you help me to understand why the test cases has failed so that i can fix them.

**Comment by ngoyal16** [comment]
can you please provide the steps to do it.?

**Comment by jamengual** [comment]
can you see the failing checks? you can google the error, sorry I'm on my phone

**Comment by ngoyal16** [comment]
@jamengual i have tried all solution found on google but no luck

**Comment by jamengual** [comment]
run 
```make init
make readme
terraform fmt
``` 

and then add and commit the files.

**Comment by ngoyal16** [comment]
@jamengual please check now

**Comment by jamengual** [comment]
to fig the issue you need to run one more command :

 make init
 make github/init

**Comment by ngoyal16** [comment]
@jamengual  i have updated the code as pre requirement.

**Comment by ngoyal16** [comment]
@goruha what could be the reason of failure?

**Comment by dudymas** [comment]
It looks like [VPC limit](https://github.com/cloudposse/actions/actions/runs/4301391120/jobs/7498567681#step:9:1841) is the issue

**Comment by ngoyal16** [comment]
@jamengual  can you please try again

**Comment by jamengual** [comment]
@ngoyal16 I ping CloudPosse to see if they can bump up the vpc count.

**Comment by ngoyal16** [comment]
@jamengual  thank you for your effort.

**Comment by ngoyal16** [comment]
@jamengual any update on same?

File: main.tf [hunk 1]
```diff
@@ -394,3 +394,11 @@ resource "aws_appautoscaling_policy" "replicas" {
     scale_out_cooldown = var.autoscaling_scale_out_cooldown
   }
 }
+
+resource "aws_rds_cluster_activity_stream" "primary" {
+  count = local.enabled && var.activity_stream_enabled ? 1 : 0
+
+  resource_arn = aws_rds_cluster.primary[0].arn
```
Comment (nitrocode) [inline main.tf:401]:
```suggestion
  resource_arn = one(aws_rds_cluster.primary[*].arn)
```

**Review by nitrocode — DISMISSED** [review]

**Comment by ngoyal16** [comment]
@goruha or @jamengual can you please retest?

**Comment by joe-niland** [comment]
@ngoyal16 the VPC limit issue still needs to be addressed

**Review by Nuru — APPROVED** [review]
Restoring @nitrocode's stale approval

**Pull request merged**